lib/mongo_client.js

```diff
@@ -531,7 +534,6 @@ MongoClient.prototype.isConnected = function(options) {
  * @param {boolean} [options.promoteLongs=true] Promotes long values to number if they fit inside the 53 bits resolution
  * @param {boolean} [options.domainsEnabled=false] Enable the wrapping of the callback in the current domain, disabled by default to avoid perf hit
  * @param {boolean|function} [options.checkServerIdentity=true] Ensure we check server identify during SSL, set to false to disable checking. Only works for Node 0.12.x or higher. You can pass in a boolean or your own checkServerIdentity override function
- * @param {object} [options.validateOptions=false] Validate MongoClient passed in options for correctness
```

Comment:
I think we still need to document this, even if we are deprecating it. You support it's value below.